modules/dxp/apps/search-experiences/search-experiences-rest-impl/rest-openapi.yaml

```diff
@@ -1022,6 +1022,31 @@ paths:
                             schema:
                                 type: string
             tags: ["SXPBlueprint"]
+    /sxp-blueprints/{sxpBlueprintId}/set-as-collection-provider:
```

Comment:
wondering if we can just put this under /sxp-blueprints/{sxpBlueprintId} as post? that endpoint already has delete, get, patch, put... so maybe there is room for this post?

Reply:
I don't understand your suggestion, perhaps what you mean is that we could just save a Blueprint with generalConfiguration.collectionProvider set to true instead of having this endpoint?

Reply:
I worked with @oliv-yu to remove the REST endpoint.